sdk/storage/README.md

```diff
@@ -103,13 +103,13 @@ npm install @azure/storage-blob@12.0.0-preview.5
 In your TypeScript or JavaScript file, import via following:
 
 ```JavaScript
-import * as Azure from "@azure/storage-blob";
+import * as AzureStorageBlob from "@azure/storage-blob";
 ```
 
 Or
 
 ```JavaScript
-const Azure = require("@azure/storage-blob");
+conss AzureStorageBlob = require("@azure/storage-blob");
```

Comment:
As long as the individual code samples have the import/require statement, I believe we dont this one specifically.

Any experienced JS/TS developer would know how to import the package.
Others would gravitate towards the code sample that already have the import/require statement. 

This line is not helpful especially if people try to combine it with the code samples in this file as the import name wont match.

@bterlson and I discussed about this offline and he agrees

Reply:
We can keep the current PR as is and log a separate issue to track the need to remove the above after consulting with @XiaoningLiu and @jiacfan

Reply:
I made an issue here: https://github.com/Azure/azure-sdk-for-js/issues/5821